src/vs/gitpod/browser/workbench/workbench.ts

```diff
@@ -946,6 +946,7 @@ async function doStart(): Promise<IDisposable> {
 		},
 		settingsSyncOptions: {
 			enabled: true,
+			extensionsSyncStateVersion: info.instanceId,
```

Comment:
Found this setting which is used to reset the the sync state if the version changes, this should happen when a workspace is restarted, so I'm using `info.instanceId` which is different on workspace restart